docker-compose-junit-jupiter/src/main/java/com/palantir/docker/compose/DockerComposeExtension.java

```diff
@@ -32,7 +32,12 @@
 
     @Override
     public void beforeAll(ExtensionContext unused) throws IOException, InterruptedException {
-        before();
+        try {
+            before();
+        } catch (RuntimeException e) {
+            after();
+            throw e;
+        }
```

Comment:
I would avoid having behaviour like this in junit5 version only. Instead this should live in `DockerComposeManager.before()` so both junit4 and 5 have the same behaviour